src/lib/icon/README.md

```diff
@@ -85,6 +85,19 @@ Multiple icon sets can be registered in the same namespace. If you request an ic
 
 Note that all SVG icons are fetched via XmlHttpRequest, and due to the same-origin policy their URLs must be on the same domain as the containing page, or their servers must be configured to allow cross-domain access.
 
+### Theming
+
+Icons can be themed to match your "primary" palette, your "accent" palette, or your "warn" palette using the `color` attribute.
+Simply pass in the palette name.
+
+Example:
```

Comment:
nit: Just to be consistent with the rest of the document, "Example:" can come at the end of the previous paragraph rather than a new one